requirements/coverage.txt

```diff
@@ -1,8 +1,36 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile requirements/coverage.in
+#    pip-compile --allow-unsafe requirements/coverage.in
```

Comment:
These changes to the comments at the tops of the `requirements/*.txt` files are due to them having been recompiled with `make requirements` which uses a slightly different `pip-compile` command than `hdev requirements` did.